boa/src/builtins/bigint/mod.rs

```diff
@@ -173,9 +173,9 @@ impl BigInt {
     pub(crate) fn as_int_n(_this: &Value, args: &[Value], ctx: &mut Context) -> Result<Value> {
         let (modulo, bits) = Self::calculate_as_uint_n(args, ctx)?;
 
-        if bits > 0 && modulo >= BigInt::from(2).pow(&BigInt::from(bits as i64 - 1)) {
+        if bits > 0 && modulo >= BigInt::from(2).pow(&BigInt::from(bits as i64 - 1)).unwrap() {
```

Comment:
Coundn't you use ? instead of .unwrap() since the function returns a Result?

Reply:
This will always succeed as the exponent is known to be positive (`bits > 0`) so I decided to simply unwrap instead mapping the `Err(String)` to an `Err(Value)` and then use `?`.

Reply:
Could you please replace the .unwrap() with an .expect() - same outcome but it includes an error message

Reply:
Done